cincinnati/src/plugins/internal/graph_builder/github_openshift_secondary_metadata_scraper/plugin.rs

```diff
@@ -28,6 +27,23 @@ pub struct GithubOpenshiftSecondaryMetadataScraperSettings {
     output_whitelist: Vec<String>,
 }
 
+impl GithubOpenshiftSecondaryMetadataScraperSettings {
+    /// Validate plugin configuration and fill in defaults.
+    pub fn deserialize_config(cfg: toml::Value) -> Fallible<Box<dyn PluginSettings>> {
+        let settings: Self = cfg.try_into()?;
+
+        ensure!(!settings.github_org.is_empty(), "empty github_org");
+        ensure!(!settings.github_repo.is_empty(), "empty github_repo");
+        ensure!(!settings.branch.is_empty(), "empty branch");
+        ensure!(
+            !settings.output_whitelist.is_empty(),
+            "empty output_whitelist"
```

Comment:
Lets add help text for `DEFAULT_OUTPUT_WHITELIST` in `cincinnati/src/plugins/internal/graph_builder`

Reply:
Do you mean that it can't be an empty string?

Reply:
I think we need a ticket to ensure proper plugin documentation, as this is not a single-case of lacking docstrings. I think the correct place to put documentation for this check would be on the `pub struct GithubOpenshiftSecondaryMetadataScraperSettings` itself. I added docstring for this there, but we should still follow-up with a more general pass over all plugin settings.

Reply:
Oh, okay, lets defer that